pkg/cluster/builder.go

```diff
@@ -117,3 +127,40 @@ func (b FileSpecBuilder) getBundles() (*releasev1.Bundles, error) {
 
 	return bundles.Read(b.reader, bundlesURL)
 }
+
+func (b FileSpecBuilder) getRelease(config *Config) (*releasev1.EksARelease, error) {
+	var opts []manifests.ReaderOpt
+	if b.releasesManifestURL != "" {
+		opts = append(opts, manifests.WithReleasesManifest(b.releasesManifestURL))
+	}
+	manifestReader := manifests.NewReader(b.reader, opts...)
+	return manifestReader.ReadReleaseForVersion(string(*config.Cluster.Spec.EksaVersion))
+}
+
+func getEKSARelease(release *releasev1.EksARelease, bundle *releasev1.Bundles, cluster *v1alpha1.Cluster) *v1alpha1.EKSARelease {
```

Comment:
this method signature is a bit weird
what about something like `buildEKSARelease(release, bundles)